app/src/main/res/values/untranslatableStrings.xml

```diff
@@ -8,4 +8,14 @@
     <string name="quest_postboxRoyalCypher_type_gr" translatable="false">G R</string>
     <string name="quest_postboxRoyalCypher_type_eviir" translatable="false">E VII R</string>
     <string name="quest_postboxRoyalCypher_type_vr" translatable="false">V R</string>
+
+    <string name="quest_policeType_type_carabinieri" translatable="false">Carabinieri</string>
+    <string name="quest_policeType_type_guardia_costiera" translatable="false">Guardia Costiera</string>
+    <string name="quest_policeType_type_guardia_di_finanza" translatable="false">Guardia di Finanza</string>
+    <string name="quest_policeType_type_polizia_di_stato" translatable="false">Polizia di Stato</string>
+    <string name="quest_policeType_type_polizia_municipale" translatable="false">Polizia Municipale</string>
+    <string name="quest_policeType_type_polizia_locale" translatable="false">Polizia Locale</string>
+    <string name="quest_policeType_type_polizia_stradale" translatable="false">Polizia Stradale</string>
+    <string name="quest_policeType_type_polizia_ferroviaria" translatable="false">Polizia Ferroviaria</string>
+    <string name="quest_policeType_type_polizia_penitenziaria" translatable="false">Polizia Penitenziaria</string>
```

Comment:
Maybe we should include the country code in the string names, to make future extension to other countries easier? For example:

```diff
- quest_policeType_type_carabinieri
+ quest_policeType_type_it_carabinieri
```

Reply:
I agree with you, will add